Formula/shc.rb

```diff
@@ -2,7 +2,7 @@ class Shc < Formula
   desc "Shell Script Compiler"
   homepage "https://neurobin.github.io/shc"
   url "https://github.com/neurobin/shc/archive/4.0.3.tar.gz"
-  sha256 "7d7fa6a9f5f53d607ab851d739ae3d3b99ca86e2cb1425a6cab9299f673aee16"
```

Comment:
actually if you download `https://github.com/neurobin/shc/archive/refs/tags/4.0.3.tar.gz`, the sha is indeed `7d7fa6a9f5f53d607ab851d739ae3d3b99ca86e2cb1425a6cab9299f673aee16`

it is just because of the artifact url issue
![image](https://user-images.githubusercontent.com/1580956/214486348-41cf2564-fd2f-4f72-ad6f-a0680980fd3a.png)